libs/shared/ui/src/lib/components/tabs/tabs.tsx

```diff
@@ -0,0 +1,41 @@
+import { Link } from 'react-router-dom'
+
+export interface TabsItem {
+  icon: React.ReactNode
+  name: string
+  active?: boolean
+  link: string
+}
+
+export interface TabsProps {
+  items: TabsItem[]
+  contentRight?: React.ReactNode
+}
+
+export function Tabs(props: TabsProps) {
+  const { items = [], contentRight } = props
+
+  return (
+    <div className="w-full h-14 bg-white flex justify-between items-center rounded-b">
+      <div className="flex gap-1 h-14 pl-4">
+        {items.map((item: TabsItem, index: number) => (
+          <Link
+            to={item.link}
+            key={index}
+            className={`tab__item--${
```

Comment:
`tab__item--${item?.active ? 'active' : 'noactive'}` need to be removed, it's useless?